examples/react-map-gl-draw/app.js

```diff
@@ -97,6 +98,7 @@ export default class App extends Component {
             this.setState({ selectedFeatureIndex: selected && selected.selectedFeatureIndex });
           }}
           mode={modeHandler}
+          modeConfigs={modeConfigs}
```

Comment:
`EditableGeoJsonLayer` uses `modeConfig` instead of `modeConfigs`. Can we make it the same?